apps/ios/Shared/Views/Helpers/ImagePicker.swift

```diff
@@ -7,44 +7,102 @@
 //
 
 import SwiftUI
+import PhotosUI
 
-struct ImagePicker: UIViewControllerRepresentable {
+enum ImageSelectionMethod: Equatable {
+    case gallery
```

Comment:
they call it photoLibrary - should we?

Reply:
I'm tempted to leave it as library or gallery to differentiate it from the iOS versions since it is a different (self-rolled) API